pyproblems/factorial_number.py

```diff
@@ -0,0 +1,14 @@
+''' This program returns the factorial of the number passes'''
+from pyproblems.utility import is_int
+def factorial_num(number):
```

Comment:
have you tested this code for 0!

Reply:
Yes It has a test case for 0 in the test_factorial.py file

Reply:
Check the spelling of "arguement". It is should be "argument".